apps/agent/app/src/components/Transactions.js

```diff
@@ -257,11 +257,9 @@ const Transactions = React.memo(function Transactions({
 
           const { symbol, decimals } = tokenDetails[toChecksumAddress(token)]
           const formattedAmount = formatTokenAmount(
-            amount,
-            isIncoming,
+            isIncoming ? amount : -Math.abs(amount),
```

Comment:
Oh interesting, do we only have a BN object for `amount` at this moment? Otherwise, we should use `amount.neg()`.